lessonbuilder/tool/src/webapp/js/announcements.js

```diff
@@ -36,23 +36,40 @@ function showAnnouncements(url, tool_href, number, announcementsDiv){
 						var min = date.getMinutes() < 10 ? '0' + date.getMinutes() : date.getMinutes();
 						//using javascript's toLocaleDateString() to include user's locale and local time zone
 						date_time = hour +":"+min+ " " + date.toLocaleDateString();
-						text_for_announcements += '<li class="itemDiv announcementSummaryItem">';
```

Comment:
I feel all of this logic is just wrong, it should be done in the endpoint?

Reply:
Maybe, but the endpoint it is a very complex code

Reply:
I will try to doing in the endpoint